elfeed-score-adaptive.el

```diff
@@ -82,9 +82,11 @@
   (elfeed-score-adaptive-add-rules 'show entry)
   )
 
-(defun elfeed-score-adaptive-read ()
-  (let ((entry (car (elfeed-search-selected))))
-    (elfeed-score-adaptive-add-rules 'read entry)
+(defun elfeed-score-adaptive-untag-unread (entries tags)
+  (when (eq tags 'unread)
```

Comment:
This matches for the "show" case, which of course also wants to untag `'unread`. So I need something that distinguishes between these cases. This hook may not work correctly for this. Maybe there is something else that be hooked or advised.